src/Network/Network/Models/PSExtendedLocation.cs

```diff
@@ -0,0 +1,34 @@
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+namespace Microsoft.Azure.Commands.Network.Models
```

Comment:
this class should look like this:
namespace Microsoft.Azure.Commands.Network.Models
{
    public class PSExtendedLocation
    {
        public string Name { get; set; }
        public string Type { get; set; }
    }
}

Reply:
i had a strange failure occuring when i included type. lets talk when youre in the office